services/121-service/src/debit-cards-intersolve-visa/debit-cards-intersolve-visa.controller.ts

```diff
@@ -195,4 +196,56 @@ export class DebitCardsIntersolveVisaController {
       programId,
     );
   }
+
+  @AuthenticatedUser({ isAdmin: true })
+  @ApiOperation({
+    summary: 'Link a physical debit card to a registration',
+  })
+  @ApiParam({ name: 'programId', required: true, type: 'integer' })
+  @ApiParam({ name: 'referenceId', required: true, type: 'string' })
+  @ApiParam({ name: 'tokenCode', required: true, type: 'string' })
+  @ApiResponse({
+    status: HttpStatus.OK,
+    description: 'Card linked',
+  })
+  @Post(
+    'programs/:programId/registrations/:referenceId/fsps/intersolve-visa/link-card/:tokenCode',
+  )
+  public async linkDebitCardToRegistration(
+    @Param('programId', ParseIntPipe) programId: number,
+    @Param('referenceId') referenceId: string,
+    @Param('tokenCode') tokenCode: string,
+  ): Promise<void> {
+    return await this.debitCardsIntersolveVisaService.linkDebitCardToRegistration(
+      referenceId,
+      programId,
+      tokenCode,
+    );
+  }
+
+  @AuthenticatedUser({ isAdmin: true })
+  @ApiOperation({
+    summary: 'Replace a physical debit card for a registration',
+  })
+  @ApiParam({ name: 'programId', required: true, type: 'integer' })
+  @ApiParam({ name: 'referenceId', required: true, type: 'string' })
+  @ApiParam({ name: 'tokenCode', required: true, type: 'string' })
+  @ApiResponse({
+    status: HttpStatus.OK,
+    description: 'Card replaced',
+  })
+  @Post(
+    'programs/:programId/registrations/:referenceId/fsps/intersolve-visa/replace-card/:tokenCode',
+  )
+  public async replaceCard(
+    @Param('programId', ParseIntPipe) programId: number,
+    @Param('referenceId') referenceId: string,
+    @Param('tokenCode') tokenCode: string,
+  ): Promise<RegistrationEntity> {
```

Comment:
Why does this return a RegistrationEntity and not void? Feel free to disregard this comment if this was already the case